src/main/generated/io/vertx/ext/mongo/FindOptionsConverter.java

```diff
@@ -59,5 +64,8 @@ public static void toJson(FindOptions obj, java.util.Map<String, Object> json) {
     if (obj.getSort() != null) {
       json.put("sort", obj.getSort());
     }
+    if (!obj.getHint().isEmpty()) {
```

Comment:
It looks like you have modified this file manually. In this case, can you please run `mvn compile` and commit the generated files instead? Thank you